### PR TITLE
Bugfix/3.7.0/improve contributing/quac

### PR DESCRIPTION
**Pull request by quac88**
### Identify the Bug

We have several broken links and outdated instructions.

Link to the issue describing the bug that you're fixing.

[If there is not yet an issue for your bug, please open a new issue and then link to that issue in your pull request.
Note: In some cases, one person's "bug" is another person's "feature." If the pull request does not address an existing issue with the "bug" label, the maintainers have the final say on whether the current behavior is a bug.](https://github.com/opentensor/bittensor/blob/master/CONTRIBUTING.md#how-do-i-submit-a-good-enhancement-suggestion)

-->

### Alternate Designs

We will need to collectively explore cleaner and more explicit designs moving forward for our documentation.

### Possible Drawbacks

No possible side effects. This is just a markdown file.

### Release Notes

<!--

N/A

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -68,11 +73,11 @@ Provide more context by answering these questions:
 
 Include details about your configuration and environment:
 
-* **Which version of Bittensor are you using?** You can get the exact version by checking for `__version__` in `[bittensor/bittensor/__init.py`](https://github.com/opentensor/bittensor/blob/master/bittensor/__init__.py#L9).
+* **Which version of Bittensor are you using?** You can get the exact version by checking for `__version__` in `[bittensor/bittensor/__init.py`](https://github.com/opentensor/bittensor/blob/9443c66f8c230b0984e5ae423026f514d51f763a/bittensor/__init__.py#L26).
```
Comment (eduardogr) [inline CONTRIBUTING.md]: why do you change this form `master` to this commit???

Comment (quac88) [reply CONTRIBUTING.md]: the previous link was linked to the wrong line in the __init__.py file

File: DEVELOPMENT_WORKFLOW.md [hunk 2]
```diff
@@ -2,97 +2,97 @@
...
-- May branch off from: `nobunaga`
-- Must merge back into: `nobunaga`
-- Branch naming convention:
-    - Anything except master, nobunaga, nakamoto, release/* or hotfix/*
-    - Suggested: `feature/<ticket>/<descriptive-sentence>`
+    When implementing new features, hotfixes, bugfixes, or upgrades, you must adhere to a STRICT naming and merging convention.
+
+**Branch naming and merging convention:**
 
+
```
Comment (eduardogr) [inline DEVELOPMENT_WORKFLOW.md:40]: i'm missing naming convention and intructions for feature branches. We should expose it as we have for release branches

File: CONTRIBUTING.md (hunk 1 above)
Comment (eduardogr) [reply CONTRIBUTING.md]: that's great, we should keep the reference to the master branch though

**Pull request closed without merging**